lib/floe/workflow/runner/docker.rb

```diff
@@ -60,6 +63,62 @@ def output(runner_context)
           runner_context["output"] = output
         end
 
+        def wait(contexts = [], timeout: nil, since: Time.now.utc, events: [], &block)
+          refs              = Array(contexts).map { |ctx| ctx["container_ref"] }
+          container_filters = refs.map   { |container_id| [:filter, "container=#{container_id}"] } + [[:filter, "type=container"]]
```

Comment:
Minor perf improvement:

```suggestion
          container_filters = refs.map   { |container_id| [:filter, "container=#{container_id}"] } << [:filter, "type=container"]
```